database/mssql/scripts/dbscripts/PSP_PIMS_S32_00/Build/74_DML_PIMS_CLAIM.sql

```diff
@@ -61,4 +61,17 @@ GO
 INSERT [dbo].[PIMS_CLAIM] ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID]) VALUES (24, N'38028c27-0a87-4875-a5d4-40879d13cfd6', N'researchfile-edit', N'38028c27-0a87-4875-a5d4-40879d13cfd6', N'Ability to edit existing research files.', 0, 3, CAST(N'2022-04-12T02:17:49.903' AS DateTime), N'SEED', NULL, N'SEED', CAST(N'2022-04-12T07:42:25.890' AS DateTime), N'service', N'1b93f614-91da-4b32-b36e-bd2c6ebd12e2', N'', CAST(N'2022-04-12T02:17:49.903' AS DateTime), N'SEED', CAST(N'2022-04-12T07:42:25.873' AS DateTime), N'SEED')
```

Comment:
@dfilteau note that some changes made in 31_01 seem to have been overwritten in 32_00. I've added those changes back, but wanted to let you know so that the same thing doesn't happen in 33_00